gnome-extra/chrome-gnome-shell/chrome-gnome-shell-10-r1.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
```

Comment:
Did you try with python3.7? 

Also usually version bump comes before cleaning old versions, but I dont think it matters in this case since you leave at least one ebuild to tree before verbump. Just keep it in mind.

Reply:
> Did you try with python3.7?

No, I did not.